src/store/configureStore.js

```diff
@@ -0,0 +1,15 @@
+import { createStore, combineReducers } from "redux";
+import eventsReducer from "../reducers/events";
+// import filtersReducer from "../reducers/filters";
+
+export default () => {
+  const store = createStore(
+    combineReducers({
+      events: eventsReducer,
+      //   filters: filtersReducer,
+    }),
+    window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__(),
+  );
+
+  return store;
```

Comment:
Hm... instead of declaring default function which will initialize `store` for you, maybe just initialize `store` here and just export it?